src/components/SignUpForm/index.jsx

```diff
@@ -2,38 +2,57 @@ import React from "react";
 import { useForm } from "react-hook-form";
 import Joi from "joi";
 import { joiResolver } from "@hookform/resolvers/joi";
+import { useTranslation } from "react-i18next";
 import InputErrorMessage from "../InputErrorMessage";
 import { getErrorClass } from "../../utils/formHelpers";
 import { EMAIL_ALREADY_TAKEN_ERROR } from "../../utils/authHelpers";
 import "./index.scss";
 
-const schema = Joi.object({
-  firstName: Joi.string().required().messages({
-    "string.empty": `Please provide your first name`,
-  }),
-  lastName: Joi.string().required().messages({
-    "string.empty": `Please provide your last name`,
-  }),
-  email: Joi.string().required().email({ tlds: {} }).messages({
-    "string.empty": `Please provide your email`,
-    "string.email": `Please provide a valid email`,
-  }),
-  password: Joi.string().required().min(8).max(64).strict().messages({
-    "string.empty": `Please provide a password`,
-  }),
-  confirmPassword: Joi.string()
-    .valid(Joi.ref("password"))
-    .required()
-    .strict()
-    .messages({
-      "any.only": "Those passwords didn't match",
-    }),
-  acceptTerms: Joi.valid("accepted").required().messages({
-    "any.only": `You must accept our terms of service and privacy policy`,
-  }),
-});
-
 export default function SignUpForm({ submit }) {
+  const { t } = useTranslation();
+
+  const schema = Joi.object({
```

Comment:
what is this

Reply:
I moved this object into the compeonent to be able to use `t` hook.